.cirrus.yml

```
@@ -1,5 +1,14 @@
 task:
-  trigger_type: manual
+  # On Cirrus CI, cron tasks does not start if it has
+  # manual trigger type enabled - expected behaviour
+  # see: https://github.com/cirruslabs/cirrus-ci-docs/discussions/949#discussioncomment-1853964
+  matrix:
+    # Keep manual trigger for all pushes (prs and merges)
+    - trigger_type: manual
+      only_if: $CIRRUS_CRON == ''
+    # Only trigger task automatically if it's a CRON and main branch
```

```suggestion
    # Only trigger task automatically if it's a cron and main branch
```
`cron` is not an initialism, as far as I know, and `man cron` doesn't indicate it should be upper case.